### ISSUE TITLE
Block color doesn‘t change after winter

### ISSUE THREAD

**Issue by ghost** [lucaargolo/fabric-seasons#134]
I host a server with this mod included. We just experienced our first winter. However, now that we’re in spring and snow has melted a lot of the grass/podzol/mycelium blocks still have the gray texture as if they have snow on top of it. This is server wide. We removed the mod server and client side but this doesn‘t fix the issue. How can we fix this without manually using the fill command to replace those blocks …. absolutely everywhere?

**Comment by sak1rro** [comment]
I have the same problem, if there are not too many blocks, try setting fire to each white block with a lighter to update the texture, and then put out the fire

**Comment by ginsm** [comment]
Having the same issue. Have either of you figured this out? 🤔

**Comment by ghost** [comment]
Nope, we've resorted to using the fill command to replace the blocks wherever it's truly bothering us, otherwise... guess we gotta live with it for now.
